src/Extension/Helper.php

```diff
@@ -23,6 +23,27 @@ public static function randomElement(array $array): mixed
         return $array[array_rand($array, 1)];
     }
 
+    /**
+     * Return a boolean, true or false.
+     *
+     * @param int $chanceOfGettingTrue Between 0 (always get false) and 100 (always get true)
+     */
+    public static function boolean(int $chanceOfGettingTrue = 50): bool
+    {
+        return self::numberBetween(1, 100) <= $chanceOfGettingTrue;
+    }
+
+    /**
+     * Returns a random number between start and end
+     */
+    public static function numberBetween(int $start = 0, int $end = 2147483647): int
+    {
+        $min = min($start, $end);
+        $max = max($start, $end);
+
+        return mt_rand($min, $max);
```

Comment:
random_int should be used in place of mt_rand or rand where we can.

Reply:
Nice catch, just ported the original source at this point.

Will update later or if someone is sooner feel free todo so